source/video_core/src/core/hw/gpu.cpp

```diff
@@ -280,6 +281,8 @@ inline void Write(Pica::Context& context, u32 addr, const T data) {
                     }
 
                     for (u32 y = 0; y < scaled_output_height; ++y) {
+			u32 flipped_y = config.flip_data ? ((config.display_transfer.input_height - 1) - y) : y;
```

Comment:
Should use spaces for indentation.
```suggestion
            u32 flipped_y = config.flip_data ? ((config.display_transfer.input_height - 1) - y) : y;
```